elements-harness/src/elementd_rpc.rs

```diff
@@ -196,10 +200,10 @@ impl Client {
                 .await
                 .context("cannot unblind raw source transaction")?;
             if unblinded_raw_tx.output[source_vout as usize]
-                .as_explicit()
+                .clone()
+                .into_explicit()
```

Comment:
I see you added `to_confidential` in the upstream PR. Here is a piece of code where you could take advantage of `to_explicit`.

Reply:
Ahhhh, I forgot to add `to_explicit`, what a twit.